src/test/resources/docker/docker-compose.yml

```diff
@@ -41,6 +41,38 @@ services:
     volumes:
     - "./mysql-init.sql:/docker-entrypoint-initdb.d/mysql-init.sql"
 
+  percona-xtradb-cluster-5.7:
+    image: percona/percona-xtradb-cluster:5.7
+    ports:
+      - "33070:3306"
+    networks:
+      - cluster1_net
+    restart: always
+    environment:
+      MYSQL_ROOT_PASSWORD: LbRootPass1
+      MYSQL_DATABASE: lbcat
+      MYSQL_USER: lbuser
+      MYSQL_PASSWORD: LiquibasePass1
+      CLUSTER_NAME: cluster1
+    volumes:
+      - "./percona-xtradb-cluster-init.sql:/docker-entrypoint-initdb.d/mysql-init.sql"
```

Comment:
nit. We may name init file in containers similar to outer files. And now i got why my changes in init file didn't work - i mounted mysql init file instead 🤦‍♂️